### ISSUE TITLE
option to generate an es module instead of commonjs module

### ISSUE THREAD

**Issue by mreinstein** [pegjs/pegjs#663]
### Issue type

**Feature Request:** 

### Description

When pegjs produces an output `.js` file, it is in common js format:

```javascript
module.exports = {
  SyntaxError: peg$SyntaxError,
  parse:       peg$parse
};
```

I want the output as an es module instead:

```javascript
export { peg$parse as parse, peg$SyntaxError as SyntaxError };
```

**Comment by GreenImp** [comment]
I'm not sure, but from the PR description, this may have been resolved on the [Peggy](https://github.com/peggyjs/peggy) fork: https://github.com/peggyjs/peggy/pull/11

It looks like Peggy is gaining some momentum as an active replacement for pegjs.

**Comment by hildjj** [comment]
Yes, it's resolved in Peggy.  We're working toward a first release within days, I hope.

**Comment by GreenImp** [comment]
That's fantastic! Really nice to see this project coming back to life.

**Comment by mreinstein** [comment]
oh wow, that _is_ fantastic news. pegjs has one of the most pleasant syntaxes, so I'm really happy to see peggy breathe new life into it! Will head over there and start exploring. :)

**Comment by hildjj** [comment]
We just did a pre-release version of peggy, 1.0.0-rc.0.  Can a few of you try this out in your existing projects, please?  We'll do a full release as soon as we have a few data points.

**Comment by stoneRdev** [comment]
@hildjj I might try it, if it's suitable as a drop in replacement for PEGjs. Meaning if I don't have to change any of my grammars or actions, I'll give it a try

**Comment by mreinstein** [comment]
I've definitely been meaning to try this in my own project, just haven't gotten to it yet https://github.com/mreinstein/constraint-solver/issues/5

**Comment by hildjj** [comment]
The intent is that you just `s/pegjs/peggy/g` and nothing changes other than you get support. :)

**Comment by hildjj** [comment]
An example, on one of my projects: https://github.com/hildjj/resolv-conf/commit/e175e131cee11410cd7e52c591e126039d4b9244

**Comment by mreinstein** [comment]
I'll try to find some time this afternoon to take it for a spin, I'm definitely keen/excited to try this, just overworked at the moment. :)

**Comment by stoneRdev** [comment]
Well, as far as the online version goes, it all checks out, so consider me migrated

**Comment by mreinstein** [comment]
Same, I'm migragted to peggy!
